packages/cli/src/commands/update/workflow.ts

```diff
@@ -51,13 +51,19 @@ export class UpdateWorkflowCommand extends BaseCommand {
 		}
 
 		const newState = flags.active === 'true';
+		const action = newState ? 'Activating' : 'Deactivating';
 
 		if (flags.id) {
-			this.logger.info(`Deactivating workflow with ID: ${flags.id}`);
+			this.logger.info(`${action} workflow with ID: ${flags.id}`);
 			await Container.get(WorkflowRepository).updateActiveState(flags.id, newState);
 		} else {
-			this.logger.info('Deactivating all workflows');
-			await Container.get(WorkflowRepository).deactivateAll();
+			this.logger.info(`${action} all workflows`);
+			if (newState) {
+				// TODO: Maybe there should be an `updateActiveStateForAll(state)`?
```

Comment:
Wondering if that makes sense? What do you think?

Reply:
Since the existing `deactivateAll` is only used here, it's fine if we rename it and make it more generic so we can just use a single flag, but either work. The current naming is also very explicit and clear in what it does.